web/src/components/SideBar.tsx

```diff
@@ -41,7 +41,7 @@ const SideBar = (props: Props) => {
     } else {
       return props.errors.map((error, index) => {
         return (
-          <ListGroup.Item action className={"sidebar-item" + " " + sidebarItemClass(error)} onClick={_ => props.handleErrorSelect(error.aggregation_key)} active={ props.activeError === undefined ? false : error.aggregation_key === props.activeError.aggregation_key } key={index}>
+          <ListGroup.Item action className={"sidebar-item" + " " + sidebarItemClass(error)} onClick={_ => props.handleErrorSelect(encodeURIComponent(error.aggregation_key))} active={ props.activeError === undefined ? false : error.aggregation_key === props.activeError.aggregation_key } key={index}>
```

Comment:
what do you think about doing this in App.tsx, in the `handlerErrorSelect` function itself? that way, if we set the active error in other places using this function, it will work too

Reply:
much better 👌